bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -265,8 +287,9 @@ public ByteKeyRange toByteKeyRange() {
   @Override
   public void populateDisplayData(DisplayData.Builder builder) {
     super.populateDisplayData(builder);
-    builder
-        .add(DisplayData.item("readRowsRequest", request.toString()).withLabel("ReadRowsRequest"));
+    if (areParametersAccessible()) {
+      builder.add(
+          DisplayData.item("readRowsRequest", request.toString()).withLabel("ReadRowsRequest"));
+    }
```

Comment:
To ease debugging, maybe set the value to "Unavailable during pipeline construction" or something if the params are not accessible. It will give users something to copy/paste into a bug report

Reply:
I don't think user will see this at all at construction time.
I have confirmed this actually showed up correctly at runtime.

Reply:
I didn't mena for "Unavailable during pipeline construction"  to show up in the normal case. I meant for it show up in case something does wrong and we need to debug. It will be easier for the user to describe this message rather than the absence of it.

Reply:
Added TODO.